nixos/modules/services/networking/wireguard.nix

```diff
@@ -160,6 +160,14 @@ let
         interval of 25 seconds; however, most users will not need this.'';
       };
 
+      table = mkOption {
+        default = "main";
+        type = types.str;
+        description = ''The kernel routing table to add this peer's associated
```

Comment:
Yes, if you want to do complicated things with routing and secondaries and all sorts of things, you likely want to use the existing routing utilities. The multi-peer feature helps with "server-like" setups.

For this reason maybe you _do_ want to make that table setting per-interface?

But I'll leave it up to you.